summarization/web_entity_detection.py

```diff
@@ -175,3 +186,22 @@ def _get_top_entities(self, image_response, number_of_entities):
         return [web_entity_collection[i]["description"]
                 for i in range(number_of_entities)
                 if "description" in web_entity_collection[i]]
+
+    def _get_rgb_tuple_from_color_dict(self, color_dict):
+        return (
+            color_dict["color"]["red"],
+            color_dict["color"]["green"],
+            color_dict["color"]["blue"]
+        )
+
+    def _get_top_colors_from_image(self, image_response):
+        if "imagePropertiesAnnotation" not in image_response:
+            return [(0, 0, 0)]  # return black color
```

Comment:
Can we return `[(-1, -1, -1)]` here?

Reply:
We could but (-1, -1, -1) is an invalid color tuple and might cause errors when generator tries to read the color

Reply:
on second thought this seems valid